tests/src/TestBCTournament.cpp

```diff
@@ -111,4 +131,28 @@ void TestBCTournament::LeaveTournament()
 	TestResult tr;
 	m_bc->getTournamentService()->leaveTournament(_leaderboardId, &tr);
 	tr.run(m_bc);
+}
+
+void TestBCTournament::JoinDivision()
+{
+	TestResult tr;
+	m_bc->getTournamentService()->joinDivision(_divSetId, _tournamentCode, 100, &tr);
```

Comment:
We discussed that you should call LeaveDivision here before joining. This will avoid errors of the type:
`Cannot join division set, already joined.`